.github/workflows/gradle.yml

```diff
@@ -13,14 +13,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '8', '11', '13', '15' ]
+        java: [ '8', '8.0.192', '11', '11.0.3', '13', '15' ]
```

Comment:
What's the reason for these specific versions?

Reply:
Often vendors or customers who will use your library they'll often are on fixed versions. It's good practice for when they are in production and building on latest can and has broken things.

**GH Workflows:**
If the fixed version is green while the major version is red, you'll know the problem is with the latest version rather than the code that you've written yourself. 

**Why the distro switch?**
Adopt is no more, Temurin and Zulu are the options available instead, and Zulu provides more versions since it has been around longer than Temurin.